### PR TITLE
[zh] fix some mistakes in content/zh/docs/concepts/storage/storage-classes.md

### PR DESCRIPTION
**Pull request by CKchen0726**
```release-notes
[zh] fix some mistakes in content/zh/docs/concepts/storage/storage-classes.md
```

### CODE REVIEW

File: content/zh/docs/concepts/storage/storage-classes.md [hunk 1]
```diff
@@ -225,14 +225,14 @@ mount options specified in the `mountOptions` field of the class.
...
-the class or PV, so mount of the PV will simply fail if one is invalid.
+the class or PV, If a mount option is invalid, the PV mount fails.
  -->
 ### 挂载选项
 
 由 StorageClass 动态创建的 PersistentVolume 将使用类中 `mountOptions` 字段指定的挂载选项。
 
-如果卷插件不支持挂载选项，却指定了该选项，则制备操作会失败。
-挂载选项在 StorageClass 和 PV 上都不会做验证，所以如果挂载选项无效，那么这个 PV 就会失败。
+如果卷插件不支持挂载选项，却指定了选项，则制备操作会失败。
```
Comment (zhiguo-lu) [inline content/zh/docs/concepts/storage/storage-classes.md:234]:
```suggestion
如果卷插件不支持某挂载选项，却指定了该不受支持的选项，则制备操作会失败。
```

File: content/zh/docs/concepts/storage/storage-classes.md [hunk 2]
```diff
@@ -225,14 +225,14 @@ mount options specified in the `mountOptions` field of the class.
...
+挂载选项在 StorageClass 和 PV 上都不会做验证，所以如果有一个挂载选项无效，那么这个 PV 挂载就会失败。
```
Comment (zhiguo-lu) [inline content/zh/docs/concepts/storage/storage-classes.md:235]:
```suggestion
挂载选项在 StorageClass 和 PV 上都不会做验证，如果有任何一个挂载选项无效，那么这个 PV 挂载就会失败。
```

Comment (CKchen0726) [reply content/zh/docs/concepts/storage/storage-classes.md:235]: Your suggestion is better, I will update it.

File: content/zh/docs/concepts/storage/storage-classes.md (hunk 1 above)
Comment (CKchen0726) [reply content/zh/docs/concepts/storage/storage-classes.md:234]: This suggestion does not match the en description, I think it is better to update it after updating en description, you think?

Comment (zhiguo-lu) [reply content/zh/docs/concepts/storage/storage-classes.md:234]: 是的，你的方法很严谨。
不过英文里有很多省略，直译为中文反而不好理解。我习惯把意思补全。供你参考。比如上句，其实还是漏主语，更容易理解可以再加一个你：
如果卷插件不支持某挂载选项，你却指定了该不受支持的选项，则制备操作会失败

**Review by tanjunchen — COMMENTED** [review]
/lgtm

**Pull request merged**